test/viz.jl

```diff
@@ -0,0 +1,322 @@
+using WGLMakie, GeoMakie, GraphMakie
+using ADRIA
+using ADRIA.metrics: total_absolute_cover
+using Statistics
+
+if !@isdefined(TEST_RS)
+    const TEST_DOM, TEST_N_SAMPLES, TEST_SCENS, TEST_RS = test_rs()
+end
+
+"""Test larger scenario run with figure creation"""
+function test_rs_w_fig(rs::ADRIA.ResultSet, scens::ADRIA.DataFrame)
```

Comment:
This test wasn't specifically for vizualizations, it's intended as a high-level pass at an end-to-end analysis.

It was included because:

1. Historically we're bad at making sure analyses that we have done in the past did not break
2. We're generally bad at making sure changes did not affect visualizations.

In testing terms, this is our stand-in for an integration test.

Reply:
I'm not sure I understand what actions do you want me to take here. I can move this to `result_set.jl` (I can also do that and rename `result_set.jl`, if you have any suggestions). 

I just really think we shouldn't have them at `runtests.jl` because that makes it hard to run just the tests you are interested on when developing.

Reply:
You moved it from where it was to this `viz.jl` file. 

I don't mind moving it, but you can find a more appropriate place for it.

Reply:
I've changed the name of this file to `analysis.jl`. I don't love this name because it might look like I'm testing only the analysis module. An alternative would be to call it `end_to_end` (which is a weird name, but that's what this test is, like you said). Do you have other suggestions? @arlowhite do you have any alternative suggestions?